src/services/session.js

```diff
@@ -73,10 +73,12 @@ const getTermCode = () => {
   return `${year.toString().substr(-2)}${term}`;
 };
 
-export default {
+const sessionUtils = {
```

Comment:
Well actually this one I should have called `sessionService`, I'll update that. The other `...Utils` names were because they didn't call out to an API or anything, they were just helper functions. See e.g. `storageUtils` and `httpUtils`.